qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -57,3 +61,50 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown sample template action: %s' % action)
         self.write(result)
+
+
+def build_sample_summary(study_id, user_id, visible=None):
```

Comment:
That's correct and we believe is going to be super useful and cool. However, perhaps we can precompute that every time the templates are updated (including creation) and just display it here to make it safe. What do you think?